src/utils/Chart.js

```diff
@@ -0,0 +1,35 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    const max = d3.max(data, d => {
+      return Math.ceil(d[axis] / 1000) * 1000;
```

Comment:
If I read this right, it isn't changing the value of the data points, but setting the top and bottom values for the entire chart. Otherwise the chart would go to 569 and the top data point would be on the edge. 

The divide and multiply by x just rounds up to the nearest xth. The example rounds 569 to the nearest 1000th. If it was 1569 it would return 2000. 

If x was 100 it would round 569 to 600. if x was 10 it would round 569 to 570.

My question would be if we always want to round to 1000. What if the numbers range from 1-99?